asllib/doc/ASLSemanticsReference.tex

```diff
@@ -1508,14 +1514,15 @@ \section{SemanticsRule.PNot \label{sec:SemanticsRule.PNot}}
 \section{SemanticsRule.PRange \label{sec:SemanticsRule.PRange}}
 
     \subsection{Prose}
-   Evaluation of the pattern \texttt{p} under environment \texttt{env} is
-  \texttt{(res,new\_env)} and all of the following apply:
+  Evaluation of the pattern \texttt{p} under environment \texttt{env} with
+  respect to value \texttt{v} is \texttt{b} and all of the following apply:
     \begin{itemize}
-    \item \texttt{p} is two side-effect-free expressions \texttt{e1} and \texttt{e2};
-    \item \texttt{res} is a value \texttt{v};
-    \item \texttt{v} is greater than or equal to \texttt{e1} in \texttt{env};
-    \item \texttt{v} is lesser than or equal to \texttt{e2} in \texttt{env};
-    \item \texttt{new\_env} is \texttt{env}.
+      \item \texttt{p} is the condition of being between two side-effect-free expressions \texttt{e1} and \texttt{e2};
```

Comment:
please consider rephrasing, perhaps as follows:
of being between -> corresponding to being between greater than \texttt{e1} and lesser than \texttt{e2}
and add:
\item e1 and e2 are side-effect-free expressions